package.json

```diff
@@ -126,79 +126,7 @@
     "which": "^2.0.2",
     "write-file-atomic": "^4.0.0"
   },
-  "bundleDependencies": [
```

Comment:
This list is managed by the `scripts/bundle-and-gitignore.js` script and for now we probably wanna keep it that way.

This PR for now should be isolated to the docs change.

Reply:
Makes sense, I had miss this, thanks. It's done, I removed the non doc commit..